locations/spiders/carrefour.py

```diff
@@ -0,0 +1,75 @@
+import scrapy
+from urllib import parse
+import requests
+from locations.items import GeojsonPointItem
+
+
+MAPQUEST_KEY = 'ybe4KeKi8ACKY0eVqJXAw2QxTKnxnor8'
+
+MAPQ = 'http://www.mapquestapi.com/geocoding/v1/address?'
+
+HEADERS = {
+    'origin': 'https://www.carrefour.com.ar',
+    'accept-encoding': 'gzip, deflate, br',
+    'accept-language': 'en-US,en;q=0.8,ru;q=0.6',
+    'content-type': 'application/x-www-form-urlencoded',
+    'referer': 'https://www.carrefour.com.ar/storelocator/index',
+    'accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8'
+
+}
+
+
+class CarreFourSpider(scrapy.Spider):
+
+    name = "carrefour"
+    allowed_domains = ["www.carrefour.com.ar", 'www.mapquestapi.com', ]
+    download_delay = 1.5
+    start_urls = (
+        'https://www.carrefour.com.ar/storelocator/index/search/',
+        'https://www.carrefour.com.ar/storelocator',
+    )
+
+    def norm_time(self, timetable):
+        return ''
```

Comment:
Looks good. Just leaving this note here to remind myself to look at the hours when you finish that part.

Reply:
👍  I will work on that in the evening.

Reply:
@iandees do u want me to convert the days into English?

Reply:
Yes please.